hack/jenkins/e2e.sh

```diff
@@ -221,462 +122,336 @@ case ${JOB_NAME} in
 
   # Runs a subset of tests on GCE in parallel. Run against all pending PRs.
   kubernetes-pull-build-test-e2e-gce)
-    : ${E2E_CLUSTER_NAME:="jnks-e2e-gce-${NODE_NAME}-${EXECUTOR_NUMBER}"}
-    : ${E2E_NETWORK:="e2e-gce-${NODE_NAME}-${EXECUTOR_NUMBER}"}
-    : ${GINKGO_PARALLEL:="y"}
+    # XXX Not a unique project
```

Comment:
probably a bad idea for this PR (already confusing enough), but we should probably consistently order these env vars. maybe when we push them into YAML?